extras/asm-multicluster/README.md

```diff
@@ -126,7 +126,7 @@ transactionhistory-84cf479f65-lzgdt   2/2     Running   0          14m
 7. **Access the Bank of Anthos frontend** using the Istio IngressGateway's LoadBalancer IP.
 
 ```
-kubectx cluster-1; kubectl get svc istio-ingressgateway -n istio-system
+kubectx cluster-1; kubectl get svc istio-ingressgateway -n asm-gateways
```

Comment:
`asm-gateways` has just become the standard namespace that I've seen used since the move away from using `istio-system` but it can be anything. I can change it to `asm-ingress`